client/ayon_core/hosts/houdini/api/plugin.py

```diff
@@ -347,3 +362,45 @@ def apply_settings(self, project_settings):
 
         for key, value in settings.items():
             setattr(self, key, value)
+
+    @staticmethod
```

Comment:
Why are they static methods?

Reply:
because we don't need to pass `self`.. these methods process the given node with given data or return data.

Reply:
> because we don't need to pass self

You don't need it "now".

Reply:
> You don't need it "now".

Ouch!